### PR TITLE
IBM Z restricted network notes

### PR DESCRIPTION
**Pull request by kalexand-rh**
@sferich888, @reestr, will you PTAL? @aheslin asked that I clarify that restricted network installations are not supported on IBM Z right now.

### CODE REVIEW

**Comment by sferich888** [comment]
Correct were still working on test these procedures, until that's done we can't claim support.

**Comment by kalexand-rh** [comment]
I checked with Tim, and he said that he'd prefer it say that it's not "not tested or supported," so I've added that change.

**Comment by kalexand-rh** [comment]
@openshift/team-documentation, PTAL?

File: modules/cluster-entitlements.adoc [hunk 1]
```diff
@@ -39,7 +39,7 @@ ifdef::openshift-origin,openshift-enterprise[]
 
 [IMPORTANT]
 ====
-If your cluster cannot have direct internet access, you can perform a restricted network installation on infrastructure that you provision. During that process, you download the content that is required and use it to populate a mirror registry with the packages that you need to install a cluster and generate the installation program. With some installation types, the environment that you install your cluster in will not require internet access. Before you update the cluster, you update the content of the mirror registry.
+If your cluster cannot have direct internet access, you can perform a restricted network installation on some types of infrastructure that you provision. During that process, you download the content that is required and use it to populate a mirror registry with the packages that you need to install a cluster and generate the installation program. With some installation types, the environment that you install your cluster in will not require internet access. Before you update the cluster, you update the content of the mirror registry.
```
Comment (mburke5678) [inline modules/cluster-entitlements.adoc]: Should we use _If your cluster does note have..._ ? _cluster cannot have_ doesn't sound right.

Also, _With some installation types, the environment that you install your cluster in will not require internet access_ sounds redundant here. We've established that the cluster doesn't have internet access in the first sentence.

File: installing/install_config/installation-types.adoc [hunk 2]
```diff
@@ -6,6 +6,11 @@ toc::[]
 
 You can perform different types of installations on different platforms.
 
+[NOTE]
+====
+Not all installation options are currently available for all platforms.
```
Comment (mburke5678) [inline installing/install_config/installation-types.adoc]: Maybe a dumb question here. Are the installation options available/not available the options in the tables below? 
_Not all installation options are currently available for all platforms, as shown in the following tables_

**Comment by mburke5678** [comment]
@kalexand-rh A couple of nits. Otherwise LGTM.

File: modules/cluster-entitlements.adoc (hunk 1 above)
Comment (kalexand-rh) [reply modules/cluster-entitlements.adoc]: With some installation types, you have to have some internet access to your cluster for the APIs to work.

File: installing/install_config/installation-types.adoc (hunk 2 above)
Comment (kalexand-rh) [reply installing/install_config/installation-types.adoc]: As long as I keep the table up to date, yes. :D

**Pull request merged**